src/actions/pause.js

```diff
@@ -103,7 +105,7 @@ export function command({ type }: CommandType) {
 
     return dispatch({
       type: "COMMAND",
-      value: undefined
+      value: { type }
```

Comment:
lets add some jest unit tests to demonstrate that command is set and then cleared when the command is done.

I think we can do something like:


```js
it("lala", async () => {
  let resolve = null;
  const slowClient = {
     stepIn: () => new Promise(_resolve => {resolve = _resolve})
  }
  
  await actions.paused()
  actions.stepIn()
  
  expect(isStepping(state)).toBeTruthy()
  resolve();
  expect(isStepping(state)).toBeFalsey()
}
```